### DISCUSSION TITLE
v1.5.9

### DISCUSSION THREAD

**Discussion by cx-joao-reigota** [Checkmarx/kics#5402] [Announcements]
# 🚀 New features and improvements

added 3 queries for Kubernetes and CloudFormation
feat(resolver): added openapi file resolver for json and yaml parsers (#5396)
feat(result): added resourceType and resourceName to Kubernetes queries result (#5355)
feat(result): added resourceType and resourceName to Azure Resource Management queries result (#5356)
feat(result): added resourceType and resourceName to Google Deployment Management queries result (#5357)
feat(result): added resourceType and resourceName to Ansible queries result  (#5362)

# 🐛 Bug fixes

fix(query): fix/cmk rotation disabled on terraform asymmetric key creation (#5344) by @LupovichRan
fix(query): changed severity of Memcached Disabled query (#5349)
fix(ci): fixed access to CIFlag (#5395)
fix(cpu): fix number cpus macos (#5371)
fix(vulnerability builder): fixed and improved DefaultVulnerabilityBuilder (#5347)
fix(detector): fixed bug on GetBracketValues function (#5343)

# 📦 Dependency updates bumps

build(go): bump golang version to 1.18 (#5348)
build(deps): bump helm.sh/helm/v3 from 3.8.2 to 3.9.0 (#5374)
build(deps): bump github.com/tdewolff/minify/v2 from 2.11.2 to 2.11.5 (#5392)
build(deps): bump mvdan.cc/sh/v3 from 3.4.3 to 3.5.1 (#5341) (#5391)
build(deps): bump github.com/hashicorp/go-getter from 1.5.11 to 1.6.1 (#5372) (#5378)
build(deps): bump github.com/aws/aws-sdk-go from 1.44.11 to 1.44.21 (#5340) (#5345) (#5350) (#5353) (#5366) (#5373) (#5377) (#5385) (#5393) (#5397)
ci(deps): bump golangci/golangci-lint-action from 3.1.0 to 3.2.0 (#5339)
ci(deps): bump goreleaser/goreleaser-action from 2.9.1 to 3.0.0 (#5390)
ci(deps): bump alpine from 3.15.4 to 3.16.0 (#5394)

# 👻 Maintenance

update(resolver): implemented limit in resolver to 50 files (#5398)
update(kics): reduced the number of code files (#5325)
update(query): updated description of Ensure Administrative Boundaries Between Resources query (#5388)
updated(query): added support to .crt file for Client Certificate Authentication Not Setup Properly query (#5360)

Contributors: @LupovichRan

<hr /><em>This discussion was created from the release <a href='https://github.com/Checkmarx/kics/releases/tag/v1.5.9'>v1.5.9</a>.</em>

**Comment by andyshinn** [comment]
Is the Docker release maybe hung up? Been pushing for a while now: https://github.com/Checkmarx/kics/actions/runs/2386033707

**Comment by LupovichRan** [comment]
Last version took around 1 hour and 40 minutes for this step




Regards,


Ran Lupovich


בתאריך יום ד׳, 25 במאי 2022, 21:12, מאת Andy Shinn ‏<
***@***.***>:

**Reply by andyshinn** [reply]
Wow, that seems so long to build / push Docker image. But I guess 👍 if it is normal.

**Reply by cx-joao-reigota** [reply]
Hi @andyshinn and @LupovichRan,

Usually, the docker image takes some time to build and push but it is available now

Thank you
